arch/arm/soc/st_stm32/stm32f2/Kconfig.defconfig.stm32f207xe

```diff
@@ -0,0 +1,34 @@
+# Kconfig - ST Microelectronics stm32f207XE MCU
+#
+# Copyright (c) 2018 qianfan Zhao <qianfanguijin@163.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F207XE
```

Comment:
Add this soc in `arch/arm/soc/st_stm32/stm32f2/Kconfig.defconfig.stm32f207xx`
(cf stm32f417xx soc for instance)